marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/entities/search/EntitySearchManager.java

```diff
@@ -146,9 +139,9 @@ public StringHandle search(SearchQuery searchQuery) {
                         + "various search features.");
             }
 
-            throw new DataHubException(e.getServerMessage(), e);
+            throw new RuntimeException(e.getServerMessage(), e);
```

Comment:
Just do "throw e" here - MarkLogicServerException is a subclass of RuntimeException, so there's no need to wrap it (vs something like IOException).